languages/csharp/reference/exercise-concepts/isbn-verifier.md

```diff
@@ -0,0 +1,9 @@
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/isbn-verifier/Example.cs)
+
+ ## General
+ - functions: used as the main entry point to this exercise
+ - static functions: used as the main entry point to this exercise
+ - regex: use regex to implment the checker
+ - arrays: used to store the isbn numbers
```

Comment:
I'd personally call this `strings` even though it can technically be treated as array of characters

Reply:
I agree with the `strings` suggestion